config/ember-try.js

```diff
@@ -56,6 +56,7 @@ module.exports = async function () {
         npm: {
           devDependencies: {
             '@ember/jquery': '^1.1.0',
+            'ember-source': '~3.28.0',
```

Comment:
I thiiiink basd on the error message you'll also need `'@ember/optional-features': '~2.0.0',` here

Reply:
but i guess the suite is passing so 🤷🏾

Reply:
Up until 4.x, you just get a warning, which is why we don't need it if we pin the dependency.  Once `ember-source` is 4.x, then we need to not only include it, but also do something with the configuration.

Although, I think I have an idea.....